pandas/tests/frame/methods/test_replace.py

```diff
@@ -753,6 +753,13 @@ def test_replace_for_new_dtypes(self, datetime_frame):
                 "bar",
                 DataFrame({"dt": [datetime(3017, 12, 20)], "str": ["bar"]}),
             ),
+            # GH 36782
+            (
+                DataFrame({"dt": [datetime(2920, 10, 1)]}),
+                datetime(2920, 10, 1),  # out of range date
```

Comment:
You can remove the comments in this line and the following line

Reply:
Okay - thanks for the feedback. I'll make the change. 

Is it best practice to leave out comments like this? Is the GH reference enough? 

Do you know why the build check failed? Looks like it timed out, is there anything I should do to prevent this?